9.secretstore/components/statestore.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
We don't need this file as we are not running in standalone mode.

Reply:
Fixed